src/modules/logcollector/src/logcollector.cpp

```diff
@@ -37,13 +37,13 @@ void Logcollector::SetupFileReader(const configuration::ConfigurationParser& con
     long reloadInterval = config::DEFAULT_RELOAD_INTERVAL;
 
     try {
-        fileWait = m_enabled = configurationParser.GetConfig<long>("logcollector", "file_wait");
+        fileWait = configurationParser.GetConfig<long>("logcollector", "file_wait");
```

Comment:
Nice catch @Nicogp 

I'm negatively surprised that this wasn't catch any warning or static analysis. I played around in godbolt ( https://godbolt.org/z/jb8564oz5 ) and found that there's one warning we should have seen:

```
[warning: Although the value stored to 'enabled' is used in the enclosing expression, the value is never actually read from 'enabled' [clang-analyzer-deadcode.DeadStores]](javascript:;)
```

Something to look into later on...